toolbox/fdc3-workbench/index.html

```diff
@@ -2,19 +2,20 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+		<link rel="icon" href="/favicon.ico" />
```

Comment:
this willresolve to https://fdc3.finos.org/ when deployed which is not right, think it might need to be:
```suggestion
		<link rel="icon" href="favicon.ico" />
```

Reply:
but then it appears to be fine in the preview at https://deploy-preview-894--lambent-kulfi-cf51a7.netlify.app/toolbox/fdc3-workbench/

where it requests it from: https://deploy-preview-894--lambent-kulfi-cf51a7.netlify.app/toolbox/fdc3-workbench/favicon.ico correctly.

I am however having trouble with a stylesheet import: 
![image](https://user-images.githubusercontent.com/1701764/213772144-7b29f240-9f01-4482-9717-689c5762830d.png)

Reply:
Yes, these should magically work because of...
```json
  base: "/toolbox/fdc3-workbench",
```
https://github.com/finos/FDC3/pull/894/files#diff-3dce4da6cf904c8281b3b478625ecd2f307f4c84ecd9f32fa736a827f58cf643R7